modules/vaos/app/controllers/vaos/clinics_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_dependency 'vaos/application_controller'
+
+module VAOS
+  class ClinicsController < ApplicationController
+    def index
+      response = systems_service.get_facility_clinics(
+        current_user,
+        clinics_params[:facility_id],
+        clinics_params[:type_of_care_id],
+        clinics_params[:system_id]
```

Comment:
Thoughts on using a `Clinic` model?

Reply:
will definitely be doing this for POSTs and PUTs I think where we get a lot of data. Might do something like this as a future refactor for some of the GETs but I think it's going to decide if these types of things can go in as mixins vs poro once we've got more of them built.